### PR TITLE
Update depedencies and style. Closes #63 Closes #64

### LINKED ISSUE #63: ES2015 Updates

### LINKED ISSUE #64: Update dependencies

### PR DESCRIPTION
**Pull request by vdeturckheim**
and travis' targets.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-sudo: false
 language: node_js
 
 node_js:
-  - 0.10
-  - 4
+  - "4"
+  - "node"
```
Comment (AdriVanHoudt) [inline .travis.yml]: what does this translate to?
Shouldn't it be "4" "4.0" "5"?

Comment (vdeturckheim) [inline .travis.yml]: * "4" -> v4.4.0
* "node" -> v5.8.0
It is the same targets as hapi.  I believe it is @arb 's requirement for reporters.

Comment (AdriVanHoudt) [inline .travis.yml]: hmm I prefer the more verbose option :P
also hapi does not test on 4.0 anymore? Maybe I dreamt that :D

Comment (vdeturckheim) [inline .travis.yml]: Dreams can come true https://github.com/hapijs/hapi/blob/master/.travis.yml

Comment (Marsup) [inline .travis.yml]: @AdriVanHoudt hapi's policy is to support LTS and latest stable, that's the only way to make that happen.

Comment (AdriVanHoudt) [inline .travis.yml]: ah ok np! must have missed that change :P

**Comment by arb** [comment]
Most of the string manipulation in here should be replaced with es2015 string templates.

File: test/index.js [hunk 2]
```diff
@@ -120,37 +121,37 @@ internals.readStream = function (done) {
...
-var expect = Code.expect;
-var describe = lab.describe;
-var it = lab.it;
+const lab = exports.lab = Lab.script();
+const expect = Code.expect;
+const describe = lab.describe;
+const it = lab.it;
 
-describe('GoodConsole', function () {
+describe('GoodConsole', () => {
```
Comment (arb) [inline test/index.js]: Please add the `plan` option to every test.

Comment (vdeturckheim) [inline test/index.js]: ack

**Comment by vdeturckheim** [comment]
closing in favor of #67

**Pull request closed without merging**